Donation-platform/src/main/java/com/google/model/Users.java

```diff
@@ -0,0 +1,72 @@
+package com.google.model;
+
+import java.util.List;
+import java.util.Objects;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+
+/** Represents a Users : id, userName, email, userInterests, charitiesDonatedTo. */
+public final class Users {
+
+  private Key id;
+  private String userName;
+  private String email;
+  private List<Integer> userInterests;
```

Comment:
what is this list going to contain? why is it integer? Please add comment above each field explaining what it will contain

Reply:
It's meant to contain the tag IDs, i'll add comments and change it to the key type